src/app/dynamic/enterprise/metering/old-reports/card/component.ts

```diff
@@ -0,0 +1,36 @@
+//                Kubermatic Enterprise Read-Only License
+//                       Version 1.0 ("KERO-1.0”)
+//                   Copyright © 2020 Kubermatic GmbH
```

Comment:
```suggestion
//                   Copyright © 2022 Kubermatic GmbH
```
Please also update the year in other newly added files.